### DISCUSSION TITLE
LDM Notes for February 15th, 2023

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6970] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2023/LDM-2023-02-15.md

## Agenda

- Open questions in primary constructors
    - Capturing parameters in lambdas
    - Assigning to `this` in a `struct`

**Comment by MaceWindu** [comment]
Shold be 2023 in title

**Reply by 333fred** [reply]
You didn't see anything.
